lightning/src/ln/channelmanager.rs

```diff
@@ -9450,24 +9451,32 @@ where
 	R::Target: Router,
 	L::Target: Logger,
 {
-	fn handle_message(&self, message: OffersMessage) -> Option<OffersMessage> {
+	fn handle_message(&self, message: OffersMessage, responder: Option<Responder>) -> ResponseInstruction<OffersMessage>
+	{
```

Comment:
Keep the brace on the previous line here, too.

Reply:
Corrected in [77c393a](https://github.com/lightningdevkit/rust-lightning/commit/77c393a6eec06158cbe5fe33608fbe00e2386135)
Thanks!